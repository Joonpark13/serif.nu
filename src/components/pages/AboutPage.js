import React from 'react';

function AboutPage() {
  return (<div />);
}

export default AboutPage;
