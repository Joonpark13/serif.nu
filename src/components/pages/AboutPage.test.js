import React from 'react';
import { shallow } from 'enzyme';
import AboutPage from './AboutPage';

jest.mock('images/hero-image.jpg');
jest.mock('images/alberta.jpg');
jest.mock('images/helen.jpg');
jest.mock('images/facebook-logo.png');
jest.mock('images/joon.jpeg');
jest.mock('images/julia.jpg');
jest.mock('images/kevin.jpg');
jest.mock('images/madison.jpg');
jest.mock('images/surprised_student.svg');

describe('AboutPage', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<AboutPage />);

    expect(wrapper.get(0)).toMatchSnapshot();
  });
});
