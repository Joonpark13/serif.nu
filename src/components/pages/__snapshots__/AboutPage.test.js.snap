// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPage should render correctly 1`] = `
<div>
  <div
    className=""
  />
  <div
    className=""
  >
    <ForwardRef(WithStyles)
      color="inherit"
      variant="h3"
    >
      Simple. Fast. Visual.
    </ForwardRef(WithStyles)>
    <ForwardRef(WithStyles)
      className=""
      variant="h5"
    >
      Course planning for Northwestern University.
    </ForwardRef(WithStyles)>
  </div>
  <div
    className=""
  >
    <ForwardRef(WithStyles)
      className=""
      color="primary"
      variant="h3"
    >
      Serif.nu redefines your scheduling experience.
    </ForwardRef(WithStyles)>
    <div
      className=""
    >
      <div>
        <ForwardRef(WithStyles)
          className=""
          variant="h5"
        >
          No more drawing out your schedules on paper.
        </ForwardRef(WithStyles)>
        <ForwardRef(WithStyles)
          className=""
          variant="h5"
        >
          No more clunkly Excel spreadsheets.
        </ForwardRef(WithStyles)>
        <ForwardRef(WithStyles)
          className=""
          variant="h5"
        >
          Plan out your next quarter at Northwestern with ease and speed.
        </ForwardRef(WithStyles)>
      </div>
      <img
        alt="Student with a laptop"
        className=""
        src={Object {}}
      />
    </div>
  </div>
  <div
    className=""
  >
    <ForwardRef(WithStyles)
      className=""
      color="primary"
      variant="h3"
    >
      The Team
    </ForwardRef(WithStyles)>
    <ForwardRef(WithStyles)
      className=""
      variant="h5"
    >
      Serif.nu is run by a team of previous and current Northwestern students dedicated to providing the best scheduling experience.
    </ForwardRef(WithStyles)>
    <div
      className=""
    >
      <ForwardRef(WithStyles)
        className=""
        elevation={2}
      >
        <img
          alt="Joon Park"
          className=""
          src={Object {}}
        />
        <ForwardRef(WithStyles)
          variant="h6"
        >
          Joon Park
        </ForwardRef(WithStyles)>
        <ForwardRef(WithStyles)
          variant="subtitle1"
        >
          Creator & Team Lead
        </ForwardRef(WithStyles)>
        <ForwardRef(WithStyles)
          className=""
        >
          Software engineer at Grubhub, film composer in another life. Big fan of taking pictures that make me look like I'm falling off of things.
        </ForwardRef(WithStyles)>
      </ForwardRef(WithStyles)>
      <ForwardRef(WithStyles)
        className=""
        elevation={2}
      >
        <img
          alt="Kevin Lee"
          className=""
          src={Object {}}
        />
        <ForwardRef(WithStyles)
          variant="h6"
        >
          Kevin Lee
        </ForwardRef(WithStyles)>
        <ForwardRef(WithStyles)
          variant="subtitle1"
        >
          Lead Student Developer
        </ForwardRef(WithStyles)>
        <ForwardRef(WithStyles)
          className=""
        >
          Senior studying computer science...and nothing else! Strong enthusiast of deep dish pizza, the color turquoise, and Korean music.
        </ForwardRef(WithStyles)>
      </ForwardRef(WithStyles)>
      <ForwardRef(WithStyles)
        className=""
        elevation={2}
      >
        <img
          alt="Madison Dong"
          className=""
          src={Object {}}
        />
        <ForwardRef(WithStyles)
          variant="h6"
        >
          Madison Dong
        </ForwardRef(WithStyles)>
        <ForwardRef(WithStyles)
          variant="subtitle1"
        >
          Student Developer
        </ForwardRef(WithStyles)>
        <ForwardRef(WithStyles)
          className=""
        >
          Madison Dong is a (soon-to-be) junior studying journalism, computer science, and design, so she thinks art and tech go well together. Her three pillars of life are: adventures with friends, 7-Eleven Big Bite hot dogs, and Frank Ocean. Send her funny emails and/or articles: madisonfdong@gmail.com
        </ForwardRef(WithStyles)>
      </ForwardRef(WithStyles)>
      <ForwardRef(WithStyles)
        className=""
        elevation={2}
      >
        <img
          alt="Julia Tournant"
          className=""
          src={Object {}}
        />
        <ForwardRef(WithStyles)
          variant="h6"
        >
          Julia Tournant
        </ForwardRef(WithStyles)>
        <ForwardRef(WithStyles)
          variant="subtitle1"
        >
          Student Developer
        </ForwardRef(WithStyles)>
        <ForwardRef(WithStyles)
          className=""
        >
          Hi! Iâ€™m Julia. Iâ€™m French, currently 19 years old, and will graduate in June 2021. I study computer science and enjoy learning foreign languages. Iâ€™m also a member of the varsity golf team here at NU! You can reach out to me at juliatournant2021@u.northwestern.edu.
        </ForwardRef(WithStyles)>
      </ForwardRef(WithStyles)>
      <ForwardRef(WithStyles)
        className=""
        elevation={2}
      >
        <img
          alt="Helen Zhao"
          className=""
          src={Object {}}
        />
        <ForwardRef(WithStyles)
          variant="h6"
        >
          Helen Zhao
        </ForwardRef(WithStyles)>
        <ForwardRef(WithStyles)
          variant="subtitle1"
        >
          Student Developer
        </ForwardRef(WithStyles)>
        <ForwardRef(WithStyles)
          className=""
        >
          I am Helen, a Northwestern sophomore. Love cooking, love taekwondo and especially love watching dogs sneeze
          <span
            aria-label="sneezing face"
            role="img"
          >
             ðŸ¤§
          </span>
        </ForwardRef(WithStyles)>
      </ForwardRef(WithStyles)>
      <ForwardRef(WithStyles)
        className=""
        elevation={2}
      >
        <img
          alt="Amy Chen"
          className=""
          src={Object {}}
        />
        <ForwardRef(WithStyles)
          variant="h6"
        >
          Amy Chen
        </ForwardRef(WithStyles)>
        <ForwardRef(WithStyles)
          variant="subtitle1"
        >
          Student Developer
        </ForwardRef(WithStyles)>
        <ForwardRef(WithStyles)
          className=""
        >
          My name is Amy and I'm majoring in Statistics, Economics, and MMSS. I'm pursuing a career in software engineering. Outside of school, I love tennis and photography!
        </ForwardRef(WithStyles)>
      </ForwardRef(WithStyles)>
    </div>
  </div>
</div>
`;
